app/utils/asciidoctor.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+import { type Block, type Html5Converter } from '@asciidoctor/core'
+import { InlineConverter, loadAsciidoctor } from '@oxide/design-system/components/dist'
+import { renderToString } from 'react-dom/server'
+
+import { InlineImage } from '~/components/AsciidocBlocks/Image'
+
+const attrs = {
+  sectlinks: 'true',
+  stem: 'latexmath',
+  stylesheet: false,
+}
+
+const ad = loadAsciidoctor({})
+
+class CustomInlineConverter {
+  baseConverter: Html5Converter
+
+  constructor() {
+    this.baseConverter = new InlineConverter()
```

Comment:
This now builds on top of the custom inline converter in the design system – which itself is based on the default HTML5 converter. This is for inline conversions where the AST processor doesn't reach.